lib/amorail/entity.rb

```diff
@@ -110,15 +110,14 @@ def commit_request(attrs)
 
     def handle_response(response, method)
       return false unless response.status == 200
```

Comment:
I have added check for 204 here but it makes some specs fail.
It is because 204 response has empty body, it is used in mocks - `contact_find_stub`.

Seems like we should just return `true` in case of 204.
This way `Entity#save` would return `true` which would be AR-like.